iam/machine/README.md

```diff
@@ -0,0 +1,26 @@
+## iam/machine
+Module for creating a machine user
+
+```hcl
+module "machine_user" {
+  source  = "./machine-user.tf"
+  name    = "xqb-machine-user"
+  pgp_key = "keybase:colincoleman"
```

Comment:
For the sake of consistency, I think this variable should just be `keybase` - see here: (https://github.com/TeliaSoneraNorge/divx-terraform-modules/blob/master/iam/user/user.tf#L8-L10).

Reply:
And then we just do `"keybase:${var.keybase}" as shown in the `user.tf` link.